Resources/public/css/layout.css

```diff
@@ -355,6 +355,10 @@ body.fixed .content-header .navbar.stuck {
     border-radius: 0;
 }
 
+.box .nav-stacked > li {
```

Comment:
Not sure if it should be on `layout.css` or `styles.css`. @rande can you explain the difference?

And a special css class would be better IMO.